gtfs-realtime/proto/gtfs-realtime.proto

```diff
@@ -809,17 +851,18 @@ message TripDescriptor {
     // A trip that existed in the schedule but was removed.
     CANCELED = 3;
 
-    // Should not be used - for backwards-compatibility only.
-    REPLACEMENT = 5 [deprecated=true];
+    // A trip that replaces an existing trip in the schedule.
+    REPLACEMENT = 5;
```

Comment:
I'm still unsure of what REPLACEMENT provides more than ADDED with a DELETED trip. Is that also the same behavior as what was there in the first iteration of REPLACEMENT?

Reply:
The main advantage is that it preserves the `trip_id`, which can be used by clients to identify if a journey has been modified.

If I delete and add a new trip, the same `trip_id` cannot be used.